opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -2173,7 +2173,10 @@ namespace Opm
             const unsigned gasCompIdx = Indices::canonicalToActiveComponentIndex(FluidSystem::gasCompIdx);
             if (FluidSystem::phaseIsActive(FluidSystem::oilPhaseIdx)) {
                 const unsigned oilCompIdx = Indices::canonicalToActiveComponentIndex(FluidSystem::oilCompIdx);
-                const EvalWell rvmax = FluidSystem::gasPvt().saturatedOilVaporizationFactor(pvt_region_index, temperature, seg_pressure);
+                EvalWell rvmax = FluidSystem::gasPvt().saturatedOilVaporizationFactor(pvt_region_index, temperature, seg_pressure);
+                if (rvmax < 0.0) { // negative rvmax can happen if the seg_pressure is outside the range of the table
```

Comment:
It is good to be more specific, to say it is due to pressure lower than the table range.  Higher one will result in different symptom.